model_analyzer/config/input/config_command_profile.py

```diff
@@ -217,7 +221,8 @@ def _add_repository_configs(self):
                 field_type=ConfigPrimitive(str),
                 default_value=DEFAULT_OUTPUT_MODEL_REPOSITORY,
                 flags=['--output-model-repository-path'],
-                description='Output model repository path used by Model Analyzer.'
+                description=
+                'Output model repository path used by Model Analyzer.'
```

Comment:
Are you using the `format.py` for formatting the code? It looks like you make some formatting changes and Jacky again reverts them. Can you please check whether the code is formatted using `format.py` or not?

Reply:
Yes, I thought that the yapf version used was the same but its not, I will address this.

Reply:
The best thing to do for this might be to have a separate pull request that formats `model_analyzer/*` and `test/*` with format.py, because we'll end up with far more files changed than this PR.